mmeds/server.py

```diff
@@ -1029,6 +1030,8 @@ def sign_up(self, username, password1, password2, email):
         :password2: Second copy of the user's desire password, must match :password1:.
         :email: The email address the user is signing up with.
         """
+        # ADDING USERS FROM SERVER DISALLOWED
+        return False
```

Comment:
here and elsewhere, create a flag: prod_live even if it's just a global variable in the file, so that we can toggle it on and off more easily